backend/ethereum/channel/adjudicator_test.go

```diff
@@ -33,6 +34,128 @@ import (
 	pkgtest "perun.network/go-perun/pkg/test"
 )
 
+// TestAdjudicator tests the adjudicator.
+func TestAdjudicator(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), defaultTestTimeout)
+	defer cancel()
+
+	rng := pkgtest.Prng(t)
+	numParts := 2 + rng.Intn(maxNumParts-2)
+	s := test.NewSetup(t, rng, numParts, blockInterval)
+	a := newAdjudicatorSetup(s)
+	channeltest.TestAdjudicator(ctx, t, a)
+}
+
+type adjudicatorSetup struct {
+	setup *test.Setup
+}
+
+func newAdjudicatorSetup(setup *test.Setup) *adjudicatorSetup {
+	return &adjudicatorSetup{
+		setup: setup,
+	}
+}
+
+type testAdjudicator struct {
+	*test.SimAdjudicator
+}
+
+// Progress is a no-op because app channels are not supported yet.
```

Comment:
It is supported (testable with the `MockApp`), but i think it is ok to do optimistic cases first.

Reply:
Good point. Adapted the comment.